### ISSUE TITLE
Feature Request: Umbraco Cloud Listing all dependencies errors when restoring from one environment to another

### ISSUE THREAD

**Issue by TaarnStar** [umbraco/Umbraco.Cloud.Issues#228]
When trying to restore one environment to another, for example if we try to restore the DEV environment from LIVE we get dependency errors. Usually those errors come when a node is connected with another node through a property but one of the nodes is in recycle bin.

The errors always coma one at a time (first error ends the process of restoring content).

It would be very helpful that the first error doesn't end the process of restoring content but it gathers all the errors of the same type (dependency errors) into a list and only after that to end the process of restoring content.

That way we could fix those errors in one attempt instead of needing to start the restore process again to see another error.

**Comment by nul800sebastiaan** [comment]
Thanks @TaarnStar - agreed, it's not great to be stopped multiple times when trying to deploy. We have looked into this as you're not the only one asking for this feature. Unfortunately this is quite an enormous task and it we don't currently have a timeline for when we can implement this. I'll close this as a duplicate of https://github.com/umbraco/Umbraco.Cloud.Issues/issues/38 - make sure to subscribe that issue for progress updates.
